File: src/main/java/io/github/support/terminal/application/domains/bot_commands/dto/BotCommandDTO.java
package io.github.support.terminal.application.domains.bot_commands.dto;



import io.github.support.terminal.app.domains.core.action.models.actions.Action;
import lombok.Data;
import lombok.experimental.Accessors;

import java.util.Collection;

@Data
@Accessors(chain = true)
public class BotCommandDTO {
    private String id;
    private String name;
    private String cmd;
    private Action action;
    private Collection<String> botIds;
    private String state;
}

File: src/main/kotlin/io/github/support/terminal/application/domains/bot_commands/rest/BotCommandDTO.kt
package io.github.support.terminal.application.domains.bot_commands.rest

import io.github.support.terminal.application.domains.common.action.ActionDTO

data class BotCommandDTO(
        val id: String,
        val name: String,
        val cmdTemplate: String,
        val botIds: Collection<String>,
        val action: ActionDTO,
        val enabled: Boolean
)